src/test/java/org/sonatype/nexus/ci/config/ComToOrgMigratorIntegrationTest.groovy

```diff
@@ -93,22 +94,20 @@ class ComToOrgMigratorIntegrationTest
     when:
       def project = (FreeStyleProject)jenkins.jenkins.getItem('Freestyle-IQ')
       def buildStep = (IqPolicyEvaluatorBuildStep)project.builders[0]
+      def build = project.scheduleBuild2(0).get()
 
-    then: 'the fields are properly migrated'
+    then: 'the application is scanned and evaluated'
+      1 * iqClient.verifyOrCreateApplication(*_) >> true
+      1 * iqClient.scan(*_) >> new ScanResult(new Scan(), File.createTempFile('dummy-scan', '.xml.gz'))
```

Comment:
These changes are unrelated and not necessary

Reply:
> These changes are unrelated and not necessary

The tests fail when these statements are removed.  

```
Too few invocations for:

1 * iqClient.evaluateApplication('sample-app', 'build', _) >> new ApplicationPolicyEvaluation(0, 1, 2, 3, [],
          'http://server/link/to/report')   (0 invocations)

Unmatched invocations (ordered by similarity):

1 * iqClient.verifyOrCreateApplication('sample-app')
```